lib/compile_time.flow

```diff
@@ -0,0 +1,20 @@
+export {
+	// Speculative function, used to make output of computed constant values during compilation.
+	// Normally these function is eliminated from program before passing it to backends.
+	outputCt(msg: string, v: ?) -> ? {
+		v
+	}
+
+	// Speculative function, used to make forbid a compile-time computation of an exression.
+	// Normally these function is eliminated from program before passing it to backends.
+	forbidCt(v: ?) -> ? {
+		v
+	}
+
+	// Speculative function, used to make force a compile-time computation of an exression.
+	// In case a compile-time computation  fails, the error will be reported
+	// Normally these function is eliminated from program before passing it to backends.
+	assertCt(v: ?) -> ? {
```

Comment:
This will invite people to just increase compile times in general, so are we sure we want this? At least, we have to have a way to control this.